### DISCUSSION TITLE
When there are too many output results, set up a scroll bar for the output box. Provide retrieval in the output box

### DISCUSSION THREAD

**Discussion by ydmykr** [microsoft/vscode-jupyter#10735] [New Ideas]
I'm currently using Jupytor Notebook on VSC. I found it very uncomfortable when very long result come out because I have to scroll down a long way to go the the next cell, like shown in the picture.

![image](https://user-images.githubusercontent.com/100815370/177898738-1cdd4290-e87e-4569-a040-3568e3b77a4e.png)

So I was wondering if there is a way to make the result like the original Jupyter Notebook, where they have a separate scroll for the result as shown below.

![image](https://user-images.githubusercontent.com/100815370/177898762-0ba7f62b-2940-43dd-bfe1-ea47110b61e7.png)

As an additional question, anyone know how to search for certain words in the result on VSC? ctrl+f only searches for words in the coding cells and not the result.

**Comment by amunger** [comment]
Scrollable output regions would be handled in VS Code core, and there is an issue tracking that if you want to give your vote to that: https://github.com/microsoft/vscode/issues/144111

For your additional question, I think the filter feature in the search widget might be what you're looking for - you can include outputs with that.
![image](https://user-images.githubusercontent.com/2019016/178329692-d43a5101-bdfd-405d-b87a-cf7a08084081.png)
